### PR TITLE
Update README.md links

### PR DESCRIPTION
**Pull request by stasfilin**
Signed-off-by: Stanislav Filin <stasfilinmusic@gmail.com>

### What does this PR do?

Updating README.md links.
Old links not found

### How to test this PR?
Just click to links

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ If you are interested in fixing issues and contributing directly to the code bas
 
 #### Extending Eclipse Che
 - [Add a new language support. (to be provided soon)](https://www.eclipse.org/che/docs/che-7/adding-support-for-a-new-language/)
-- [Package your favorite VSCode extensions and make them available in Che.](https://www.eclipse.org/che/docs/che-7/using-a-visual-studio-code-extension-in-che/#publishing-a-vs-code-extension-into-the-che-plug-in-registry_using-a-visual-studio-code-extension-in-che)
+- [Package your favorite VSCode extensions and make them available in Che.](hhttps://www.eclipse.org/che/docs/che-7/end-user-guide/publishing-metadata-for-a-vs-code-extension/)
```
Comment (sparkoo) [inline README.md]:
```suggestion
- [Package your favorite VSCode extensions and make them available in Che.](https://www.eclipse.org/che/docs/che-7/end-user-guide/publishing-metadata-for-a-vs-code-extension/)
```

Comment (stasfilin) [reply README.md]: Thanks, fixed

**Comment by stasfilin** [comment]
@sparkoo Do I need to update commit message from `Co-authored-by: Michal Vala <michal.vala@gmail.com>` to `Signed-off-by: Stanislav Filin <stasfilinmusic@gmail.com>`?
'Cause one of the checks didn't pass

Best Regards,
Stanislav Filin

**Comment by sparkoo** [comment]
@stasfilin Sorry for the delay with review. I think yes. You need `signed-off` to be there. You can just append it under `Co-authored`.

File: README.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ Here you can find links on how to get started with Eclipse Che:
 ### Using Eclipse Che
 Here you can find references to useful documentation and hands-on guides to learn how to get the most of Eclipse Che:
 - [Customize Che workspaces for your projects](https://www.eclipse.org/che/docs/che-7/configuring-a-workspace-using-a-devfile/)
-- [Run VSCode Extensions in Che workspaces](https://www.eclipse.org/che/docs/che-7/using-a-visual-studio-code-extension-in-che)
-- [Configure Che for your teams](https://www.eclipse.org/che/docs/che-7/building-and-running-a-custom-registry-image/)
+- [Run VSCode Extensions in Che workspaces](https://www.eclipse.org/che/docs/che-7/end-user-guide/adding-a-vs-code-extension-to-a-workspace/)
+- [Configure Che for your teams](https://www.eclipse.org/che/docs/che-7/end-user-guide/making-a-workspace-portable-using-a-devfile/)
```
Comment (sparkoo) [inline README.md:43]: You've not only fixed the link, but changed it to completely different page. I would agree that building a devfile is more important and well suited under "Using Eclipse Che", than the old link. However, I would rename the name of the link here.

Comment (sparkoo) [reply README.md:43]: maybe even this one would be better? https://www.eclipse.org/che/docs/che-7/end-user-guide/creating-and-configuring-a-new-workspace/ or both? WDYT?

Comment (nickboldt) [reply README.md:43]: Both links sounds good. Here's a replacement PR: https://github.com/eclipse/che/pull/18836

**Review by nickboldt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**